cypress/cypress.config.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
   viewportHeight: 800,
   defaultCommandTimeout: 10000,
   video: true,
-  videoUploadOnPasses: false,
```

Comment:
This config option [got removed in v13](https://github.com/cypress-io/cypress/issues/26899)